presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -60,7 +60,7 @@ statement
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
     | SHOW GRANTS
-        ON (ALL | TABLE? qualifiedName)                                #showGrants
```

Comment:
What's the expected behavior when the TABLE keyword is missing?

Is it the case that:
- "SHOW GRANTS ON TABLE xxx" only works if a table named xxx exists, but would fail for a view called xxx or some other entity we want to support in the future.
- "SHOW GRANTS ON xxx" would work for any entity named xxx (what if namespaces are not exclusive? e.g., functions vs tables)